src/client/actions/MessageCreate.js

```diff
@@ -6,7 +6,7 @@ const { Events } = require('../../util/Constants');
 class MessageCreateAction extends Action {
   handle(data) {
     const client = this.client;
-    const channel = client.channels.cache.get(data.channel_id);
+    const channel = this.getChannel(data);
```

Comment:
It might be worth adding a JSDoc info tag below mentioning that this event will only emit for DMs when `'CHANNEL'` is included within `ClientOptions#partials`

Reply:
I believe the plan is to get rid of  the channel partial and always allow messages from uncached dms? otherwise that setting would be equivalent to "receive dms" which is pointless